### PR TITLE
Add support for single keyword dataset

### PR DESCRIPTION
**Pull request by daemon**
- Implement VAD-based aligner for single keyword datasets
- Add preprocessing scripts

### CODE REVIEW

File: ww4ff/run/train.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def evaluate_engine(dataset: WakeWordEvaluationDataset, prefix: str, save: bool
     apb.add_options(opt('--model', type=str, choices=model_names(), default='las'),
                     opt('--workspace', type=str, default=str(Path('workspaces') / 'default')),
                     opt('--load-weights', action='store_true'),
-                    opt('--vocab', type=str, nargs='+', default=[' hey', 'fire', 'fox', 'kit', 'moxie', 'rexy', 'scout']),
+                    opt('--vocab', type=str, nargs='+', default=[' hey', 'fire fox']),
```
Comment (ljj7975) [inline ww4ff/run/train.py:63]: is the space between fire and fox meant to be there?

File: ww4ff/run/demo.py [hunk 2]
```diff
@@ -50,11 +50,11 @@ def _on_audio(self, in_data, frame_count, time_info, status):
         data_ok = (in_data, pyaudio.paContinue)
         self.last_data = in_data
         self._audio_buf.append(in_data)
-        if len(self._audio_buf) != 12:
+        if len(self._audio_buf) != 16:
             return data_ok
         audio_data = b''.join(self._audio_buf)
         self._audio_buf = self._audio_buf[2:]
-        arr = np.frombuffer(audio_data, dtype=np.int16).astype(np.float) / 32768
+        arr = 1.41 * (np.frombuffer(audio_data, dtype=np.int16).astype(np.float) / 32767)
```
Comment (ljj7975) [inline ww4ff/run/demo.py:57]: can you explain what this change is for?

File: ww4ff/run/train.py (hunk 1 above)
Comment (daemon) [reply ww4ff/run/train.py:63]: Yeah, just for backwards compatibility.

File: ww4ff/run/demo.py (hunk 2 above)
Comment (daemon) [reply ww4ff/run/demo.py:57]: Ugh, the 1.41 is a bug. I'll fix it.

**Pull request merged**